common/src/Microsoft.Azure.IIoT.AspNetCore.OpenApi/src/Extensions/ApplicationBuilderEx.cs

```diff
@@ -74,9 +74,9 @@ public static class ApplicationBuilderEx {
                 foreach (var info in infos) {
                     if (config.WithAuth) {
                         options.OAuthAppName(info.Title);
-                        options.OAuthClientId(config.OpenApiAppId);
-                        if (!string.IsNullOrEmpty(config.OpenApiAppSecret)) {
-                            options.OAuthClientSecret(config.OpenApiAppSecret);
+                        options.OAuthClientId(config.AppId);
```

Comment:
I would you leave the properties as they were called before.   We should have the ability to configure the swagger independent from the client auth..